File: 1. Conditional Problems/Condional Problems in C/p9.c
// write a C program to check whether a character is uppercase or lowercase alphabet.
#include<stdio.h>


int main()
{
    char ch;
    printf("Enter your character: ");
    scanf("%c", &ch);

    if (ch >= 'a' && ch <= 'z') {
        printf("%c is a lowercase letter",ch); }
    else if (ch >= 'A' && ch <= 'Z') {
         printf("%c is a uppercase letter",ch);
    }
    else {
        printf("%c is not an Alphabet",ch);
    }
    }

File: 1. Conditional Problems/Conditional Problems in Python/p9.py
#  write a python program to check whether a character is uppercase or lowercase alphabet.

ch = input("Enter Your Character: ")

if (ch.islower()):
    print(f"{ch} is lowercase letter")
elif (ch.isupper()):
    print(f"{ch} is uppercase letter")
else:
    print(f"{ch} is not an Alphabet")